tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -185,6 +185,10 @@ shader, this value can be used as desired.
 |                                        | (this is ``true`` in the Compatibility renderer,       |
 |                                        | ``false`` in Forward+ and Forward Mobile).             |
 +----------------------------------------+--------------------------------------------------------+
+| in float **CLIP_SPACE_FAR**            | Clip space far ``z`` value.                            |
```

Comment:
This isn't a global built in?

Reply:
Seems not:
![godot windows editor x86_64_hJdSNfayZw](https://github.com/user-attachments/assets/7fd0c2b4-1fbc-4e8b-8936-ba13852d830d)

Reply:
Yeah, seems like this should be fixed - https://github.com/godotengine/godot/pull/98448